test/Microsoft.AspNetCore.Hosting.Tests/Fakes/Startup.cs

```diff
@@ -7,16 +7,17 @@
 
 namespace Microsoft.AspNetCore.Hosting.Fakes
 {
-    public class Startup : StartupBase
+    public class Startup : StartupBase, IStartup
```

Comment:
This `StartupBase` is https://github.com/aspnet/Hosting/blob/dev/test/Microsoft.AspNetCore.Hosting.Tests/Fakes/StartupBase.cs#L8-L19

I preferred the more surgical fix to implement IStartup in this class but there may be better alternatives.